### PR TITLE
migrate some services from examples to openApi part 46; affects [codeclimate librariesio nexus] and securityheaders

### PR DESCRIPTION
**Pull request by chris48s**
Refs https://github.com/badges/shields/issues/9285

With nexus, we don't have passing tests for nexus v3
I have actually managed to find a couple of public nexus servers running v3:
- https://repo.tomkeuper.com/
- https://repo.unnamed.team/
but I can't work out how to get a working badge out of either of them.

With Security Headers, the tests fail because the badges themselves are broken https://github.com/badges/shields/issues/9154 I figure lets just migrate the examples here and leave that as an issue for another time.

### CODE REVIEW

**Comment by PyvesB** [comment]
> but I can't work out how to get a working badge out of either of them.

For example: 

https://pr-9933-badges-shields.fly.dev/nexus/r/me.neznamy/tab-api?server=https%3A%2F%2Frepo.tomkeuper.com&nexusVersion=3

**Comment by chris48s** [comment]
OK. Thanks for the pointer. I think I mostly figured it out :crossed_fingers:

File: services/nexus/nexus.tester.js [hunk 1]
```diff
@@ -269,7 +267,7 @@ t.create('Nexus 3 - search snapshot version valid snapshot artifact')
 
 t.create('Nexus 3 - search snapshot version for artifact without snapshots')
   .get(
-    '/s/javax.inject/javax.inject.json?server=https://nexus.pentaho.org&nexusVersion=3',
+    '/s/com.tomkeuper/spigot.json?server=https://repo.tomkeuper.com&nexusVersion=3',
```
Comment (chris48s) [inline services/nexus/nexus.tester.js:270]: This is the one I am least sure if I got right. It does appear to throw the right error message.

File: services/codeclimate/codeclimate-coverage.tester.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ t.create('test coverage when outer user repos query returns multiple items')
               },
               latest_default_branch_test_report: {
                 data: {
-                  id: '6463c8a3e3bc340001004bce',
+                  id: '65a1662cb0077b00013cb4de',
```
Comment (chris48s) [inline services/codeclimate/codeclimate-coverage.tester.js:49]: I feel like this is not a sustainable solution for this test, but I think I'm not going to deal with re-writing this test in this PR as it is a bit of an yak-shave that isn't really related to the original point of this PR.

**Review by PyvesB — APPROVED** [review]
Thanks for updating the tests, I think all is good! 👍🏻

**Pull request merged**